smoketests/test.Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM postgis/postgis:11-2.5
+
+RUN apt-get update \
+ # install newer packages from backports
+ && apt-get install -y --no-install-recommends \
+    g++ \
+    gcc \
+    libc6-dev \
+    make \
+    pkg-config \
+    libgeos-dev \
+    libleveldb-dev \
+    libprotobuf-dev \
+    osmctools \
+    postgis \
+    osmosis \
+ # install postgresql client
+ && apt-get install -y --no-install-recommends \
```

Comment:
`psql` should be already installed, why are you re-installing it?

Reply:
Nice catch, this came from a copy and paste.